docker-image-run-py/runpy

```diff
@@ -29,7 +29,14 @@ import socketserver
 import json
 import sys
 import io
-from code_runpy_backend import Struct, run_code, package_exception
+try:
+    from code_runpy_backend import Struct, run_code, package_exception
+except ImportError:
+    # When running tests
+    import os
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir)))
```

Comment:
Sorry, could you explain again why this is necessary? If the parent interpreter was able to import `relate` and `course`, why would the child interpreter not be able to do so?

Reply:
NVM. In fact, It should be like the following, so that I can use `python runpy` in console to debug:

```python
try:
    from code_runpy_backend import Struct, run_code, package_exception
except ImportError:
    try:
        # When running tests
        from course.page.code_runpy_backend import Struct, run_code, package_exception
    except ImportError:
           # When ran by "python runpy"
           import os
           sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir)))
           from course.page.code_runpy_backend import Struct, run_code, package_exception
```

Am I right?